qa/workunits/cephtool/test.sh

```diff
@@ -1831,8 +1831,10 @@ function test_mon_pg()
   expect_false ceph osd primary-affinity osd.9999 .5
   ceph osd primary-affinity osd.0 1
 
-  ceph osd pg-temp 1.0 0 1 2
-  ceph osd pg-temp 1.0 osd.1 osd.0 osd.2
+  ceph osd pool set rbd size 2
+  ceph osd pg-temp 1.0 0 1
+  ceph osd pg-temp 1.0 osd.1 osd.0
+  expect_false ceph osd pg-temp 1.0 0 1 2
```

Comment:
@liewegas Sorry, 
http://qa-proxy.ceph.com/teuthology/sage-2017-07-10_16:55:37-rados-wip-sage-testing-distro-basic-smithi/1383107/teuthology.log 
http://qa-proxy.ceph.com/teuthology/sage-2017-07-10_16:55:37-rados-wip-sage-testing-distro-basic-smithi/1383043/teuthology.log

The above two test failures are related to this change. I think the above change fixes these now.